rfcs/text/0000-tech-planning-process.md

```diff
@@ -0,0 +1,140 @@
+quarterly roadmapping process via a Technical Objectives RFC, presented during the TL+PM Planning Call.
+
+## Author and Shepherd
+
+[Arathi Mani](mailto:arathi.mani@chanzuckerberg.com)
+
+## Motivation
+
+The new Roadmaps and Planning Process, detailed in the RFC [here](https://github.com/HumanCellAtlas/dcp-community/blob/4bb18c65af5b244460cf404465b1dbc4ee0cc65e/rfcs/text/0000-roadmaps%2Bplanning.md),
+defines a process for annual roadmaps that inform quarterly release planning. TLs (Technical Leads) should contribute
+“grassroots” knowledge about the technical requirements of the DCP on a quarterly basis to guide the 
+[quarterly Release process](https://github.com/HumanCellAtlas/dcp-community/blob/4bb18c65af5b244460cf404465b1dbc4ee0cc65e/rfcs/text/0000-roadmaps%2Bplanning.md#planning-and-executing-a-release)
+via a Technical Objectives RFC. This document details the contents of the Technical Objectives RFC as well as the
+process that must occur such that this artifact is drafted and curated according to a timeline that will allow us to
+present the artifact to the quarterly TL+PM objective curation meeting.
+
+This proposed TL Quarterly Planning Process is **owned by the collection of TLs in the DCP** and is intended to
+be a live process that iterates over time as required. The first iteration of the planning process is detailed below.  
+
+Note: The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT
+RECOMMENDED" "MAY", and "OPTIONAL" in this document are to be interpreted as described in BCP 14, 
+[RFC2119](https://www.rfc-editor.org/rfc/rfc2119.txt), and [RFC8174](https://www.rfc-editor.org/rfc/rfc8174.txt) when,
+and only when, they appear in all capitals, as shown here.
+
+### User Stories
+
+As a TL of the DCP, I am able to surface important technical priorities to Product Management that will be incorporated
+into the yearly roadmap. 
+
+As a developer on the DCP, I am able to allocate enough time to work on high priority technical architectural issues
+knowing that the work will facilitate quicker development of feature requests in the long term as informed by
+the roadmap themes.
+
+## Definitions
+
+**Milestone:** a four week timeline/duration in a Release (sometimes five when a holiday is involved).
+
+**Release:** composed of three Milestones and occur on a quarterly cadence.
+
+**Technical Objective:** a high level goal that encapsulates a general area of DCP technical work that can be flexibly
+broken up by specific sub-tasks. Sub-tasks are usually represented as Epics.  
+
+**TL Roadmapping Shepherd:** Each quarter one TL acts as the “Shepherd” to make sure the TL planning process takes place
+in time to produce the expected Technical Objectives RFC for presentation to the TL+PM planning meeting. The Shepherd
+role will rotate once a quarter according to a spreadsheet drawn up post the approval of this RFC. 
```

Comment:
It'd great for the TLs on duty if they could get notified (by people or by some cron jobs) at the beginning of each Q, or at least a few weeks before their shepherd role starts.